.github/workflows/deploy-documentation.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy Documentation
+on:
+  push:
+    #branches: [ $default-branch ]
+
+jobs:
+  deploy-documentation:
```

Comment:
nitpick

My bad, these names surfaced in places I didn't at first expect.

![Screen Shot 2020-12-15 at 1 57 28 PM](https://user-images.githubusercontent.com/98017/102277763-9a28ed00-3edd-11eb-84e9-9977baadf31a.png)

Looks like it would be better as `name: Documentation` and `jobs: deploy`.

```suggestion
name: Documentation
on:
  push:
    #branches: [ $default-branch ]

jobs:
  deploy:
```

Then, it would probably also make sense to rename file to `documentation.yml`.

Reply:
Fixed w/ respect to new plugin.